common/config/azure-pipelines/jobs/fast-ci.yaml

```diff
@@ -4,6 +4,12 @@ trigger:
   - master
   - release/*
 
+variables:
+  - group: Rush Build Cache SAS Token
+  - name: isReleaseBuild
```

Comment:
im thinking we also write to the cache on the ci after a pr is merged into master

Reply:
```
  - name: shouldWriteToCache
    value: ${{ and(eq(variables['Agent.OS'], 'Linux'), in(variables['Build.Reason'], 'IndividualCI', 'Manual')) }}
```